.gitignore

```diff
@@ -3,3 +3,4 @@ coverage
 sample/dist
 node_modules
 npm-debug.log
+package-lock.json
```

Comment:
Why are you excluding `package-lock.json`?

Reply:
so, I made this PR in this repo from https://github.com/Brightspace/react-valence-ui-iframe/pull/31 as it being a PR from forked repo, it was not triggering Actions. 

Even i was not sure why was package-log excluded. Will check that and remove this change.